### PR TITLE
KULRICE-14203 - Test Failures in DemoTravelAccountMaintenanceViewPermiss...

### PR DESCRIPTION
**Pull request by shahess**
...ionAft

### CODE REVIEW

**Comment by shahess** [comment]
Pull request for master -- https://github.com/kuali/rice/pull/94

File: rice-framework/krad-sampleapp/web/src/it/java/org/kuali/rice/krad/demo/travel/account/DemoTravelAccountMaintenanceViewPermissionAft.java [hunk 1]
```diff
@@ -72,7 +72,9 @@ private void createAndRouteDoc(String user) throws Exception {
         waitAndSelectByName("newCollectionLines['document.notes'].attachment.attachmentTypeCode", "OTH");
 
         waitAndClickByXpath("//button[@title='Add a Note']");
+        assertElementPresentByXpath("//button[contains(text(),'Delete')]");
         waitForTextPresent("Bonzo!");
+        waitForTextPresent("attachment.oth");
         assertTextPresent("attachment.oth");
```
Comment (cniesen) [inline rice-framework/krad-sampleapp/web/src/it/java/org/kuali/rice/krad/demo/travel/account/DemoTravelAccountMaintenanceViewPermissionAft.java]: waitForTextPresent causes a failure if the text isn't found.  So the assertTextPresent isn't needed.

**Comment by shahess** [comment]
The pull request has been updated and is ready to be merged if all looks good.

**Comment by sonamuthu** [comment]
Looks ok.

**Pull request merged**